app/assets/stylesheets/sequence_builder.css

```diff
@@ -165,15 +167,6 @@ padding-left: 10px;
 .content-area{
   padding-top:15px;
 }
-
-/* BUTTONS */
-
-.green-button{
```

Comment:
Yes, they are replacements and I think I already got rid of the more neo-y ones. The sequence builder blocks have three different colors and I did what I could. I think I have an idea of how to make this whole thing DRYer though, which I'll try out later.